swarm/bmt/bmt_test.go

```diff
@@ -34,6 +34,13 @@ import (
 // the actual data length generated (could be longer than max datalength of the BMT)
 const BufferSize = 4128
 
+const (
+	// segmentCount is the maximum number of segments of the underlying chunk
+	// Should be equal to max-chunk-data-size / hash-size
+	// Currently set to 128 == 4096 (default chunk size) / 32 (sha3.keccak256 size)
```

Comment:
For the note: this is only true in the unencrypted case, otherwise the reference is 64 (hash+key), and segment count is 64

Reply:
I am not sure what you mean. I don't think this matters at all for encrypted/non-encrypted case. Could you elaborate?

Reply:
Please ignore my comment, I misunderstood this